CRM/Civixero/OAuth2/TokenStoreDefault.php

```diff
@@ -13,22 +13,39 @@ class CRM_Civixero_OAuth2_TokenStoreDefault implements CRM_Civixero_OAuth2_Token
 
   protected $token;
 
+  protected $connectorID;
+
+  /**
+   * Get the CiviCRM AccountSync Connector ID
+   * @return mixed
+   */
+  public function getConnectorID() {
```

Comment:
Can we return hint this?

Reply:
Here you go: https://github.com/eileenmcnaughton/nz.co.fuzion.civixero/pull/127